ansible/roles/base/tasks/main.yml

```diff
@@ -1,14 +1,12 @@
-
```

Comment:
A lot of strange spacing artifacts showed up in this file. Do you have any sense what happened? It looks like maybe you have your editor set to single space indents instead of two space indents? We should try to make sure we’re not constantly colliding. With the YAML files, I would prefer 2 spaces (not tabs). Let’s see if we can fix this before I accept the PR. If it’s my editor, I can fix it too, but either way we should double check to see what’s going on there.